test/node/tasks/StreamUpload.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import "isomorphic-fetch";
+
+import { assert } from "chai";
+import * as fs from "fs";
+
+import { StreamUpload } from "../../../src/tasks/FileUploadUtil/FileObjectClasses/StreamUpload";
+
+describe("StreamUpload", () => {
+	const fileName = "sample_image.jpg";
```

Comment:
nit: define filePath once and use it everywhere.
```suggestion
	const fileName = "sample_image.jpg";
	const filePath = `./test/sample_files/${fileName}`;
```